.github/workflows/deploy-indexer.yaml

```diff
@@ -77,6 +77,8 @@ jobs:
       RAILWAY_TOKEN: ${{ secrets.RAILWAY_TOKEN }}
     steps:
```

Comment:
Running typecheck without first installing dependencies or building the project will fail. The workflow needs to install dependencies with pnpm before running typecheck, similar to how other workflows handle dependencies.